### PR TITLE
Replace special chars in title

### PR DESCRIPTION
**Pull request by everthemore**
Interpratation of how markdown does it [here](https://github.com/gjtorikian/html-pipeline/blob/main/lib/html/pipeline/toc_filter.rb), by 

1. Converting to lower
2. Removing all punctuation except for '-'
3. Replace whitespace with '-'

I don't check for uniqueness, since two identical titles is going to be extremely rare

### CODE REVIEW

**Comment by akhmerov** [comment]
This is amazing, I don't actually understand how it works! Something to learn. I only have one remark: please also apply the filter to the speakers' corner template.

**Comment by akhmerov** [comment]
OK, I now understand how it works, and it's still cool!

**Comment by everthemore** [comment]
Thanks! :) 
See here for a timing of string.translate vs regex replacements (which I tried originally): https://stackoverflow.com/questions/265960/best-way-to-strip-punctuation-from-a-string

File: templates/speakers_corner.md.j2 [hunk 1]
```diff
@@ -23,7 +23,7 @@ All times are shown in <span id="timezone">UTC</span> timezone.
 > {{ line }}
 {%- endfor %}
 >
-> **Authors:** {{ talk.authors }}  
+> **Authors:** {{ talk.authors }}
```
Comment (akhmerov) [inline templates/speakers_corner.md.j2:26]:
```suggestion
> **Authors:** {{ talk.authors }}  
```

File: templates/speakers_corner.md.j2 [hunk 2]
```diff
@@ -56,7 +56,7 @@ There are no upcoming talks at the moment, apply [here](https://github.com/virtu
 
 {{talk.abstract}}
 
-Authors: {{ talk.authors }}  
+Authors: {{ talk.authors }}
```
Comment (akhmerov) [inline templates/speakers_corner.md.j2:59]:
```suggestion
Authors: {{ talk.authors }}  
```

**Review by akhmerov — CHANGES_REQUESTED** [review]

**Comment by everthemore** [comment]
Thanks for fixing those, still not quite sure why Atom decides to replace some but not all

**Pull request merged**